bin/builder_package

```diff
@@ -3,32 +3,34 @@
 ###########################################################################
 # Creates builder Docker image and uploads it to DockerHub https://hub.docker.com/r/mysteriumnetwork/mysterium-node-builder/
 #
-# To create a builder image, simply execute:
-#> bin/builder_package [latest]
-
-# Usage - build local image:
+# Usage example:
+#> bin/builder_package [tag..]
```

Comment:
Does `tag..` mean that there can be multiple tags?

Reply:
Exactly